src/main/java/uk/gov/companieshouse/model/validator/EndDateValidator.java

```diff
@@ -26,9 +26,13 @@ public String validateEndDate(Appeal appeal) {
         if (!continued && rawEndDate == null) {
```

Comment:
If you want to be on the safe side and check for an empty string as well you could do if (!continued && (rawEndDate == null || rawEndDate.isEmpty()) &&) This will check rawEndDate is null and whether it's an empty string

Reply:
Not a bad shout to be fair, I'll try that now